src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,25 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
-
+        Scanner scanner = new Scanner(System.in);
+        int min = 100;
+        int max = 200;
+        int number;
+        do {
+            System.out.println("Podaj liczbe");
+            number = scanner.nextInt();
+            if (number > max) {
+                System.out.println("Liczba jest za duza");
+            } else if (number < min) {
+                System.out.println("Liczba jest za mala");
+            } else if (number % 3 != 0) {
+                System.out.println("Liczba nie jest podzielna przez 3");
+            }
+        } while (number % 3 != 0 || (number < 100) || (number > 200));
```

Comment:
Skoro wprowadziłeś wcześniej min i max, to powinieneś ich tutaj użyć, ale jeszcze lepiej wprowadź flagę boolean, żeby nie sprawdzać tutaj drugi raz tych samych warunków, które sprawdziłeś już w if-elsach.